tests/tests/tier0/bluechi-daemon-reload/test_bluechi_deamon_reload.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+import logging
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.machine import BluechiControllerMachine, BluechiAgentMachine
+from bluechi_test.config import BluechiControllerConfig, BluechiAgentConfig
+
+LOGGER = logging.getLogger(__name__)
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+
+    node_foo = nodes[NODE_FOO]
+    service_file = "simple.service"
+    service_path = os.path.join("/", "etc", "systemd", "system", service_file)
+
+    node_foo.copy_systemd_service(service_file, "systemd",
```

Comment:
Shouldn't `create_file` be used instead of `copy_systemd_service`?

Reply:
If the initial daemon-reload doesn't matter (which it does not for this test), both can be used.